test/e2e/operator_expectations.go

```diff
@@ -64,12 +64,25 @@ func (tc *testConfig) ExpectClusterOperatorStatusAvailable() error {
 			glog.Errorf("error querying api for OperatorStatus object: %v, retrying...", err)
 			return false, nil
 		}
+		ready := false
 		if available := cvoresourcemerge.FindOperatorStatusCondition(clusterOperator.Status.Conditions, osconfigv1.OperatorAvailable); available != nil {
```

Comment:
@enxebre do you want to block this PR until https://github.com/openshift/machine-api-operator/pull/219 is merged?